libs/features/mas/src/buildCheckoutUrl.js

```diff
@@ -184,12 +184,12 @@ export function buildCheckoutUrl(checkoutData) {
     url = add3in1Parameters({
       url,
       modal,
-      customerSegment: customerSegment ?? items?.[0]?.customerSegment,
-      marketSegment: marketSegment ?? items?.[0]?.marketSegment,
+      customerSegment: customerSegment,
```

Comment:
Here and below we can shorten this:
`customerSegment: customerSegment,` > `customerSegment,`